frontend/src/services/effects.ts

```diff
@@ -1,3 +1,4 @@
+import * as gfilter from 'geojson-filter'
```

Comment:
do you have to use a star import if you're only importing one thing? you just call gfilter directly below.

Reply:
I am importing everything as. I mimicked what is done in the esri-extent situation